src/midiEditorCommands.cpp

```diff
@@ -994,6 +994,55 @@ void cmdMidiFilterWindow(Command *command) {
 	}
 }
 
+void maybePreviewCurrentNoteInEventList(HWND hwnd) {
+	auto focused = ListView_GetNextItem(hwnd, -1, LVNI_FOCUSED);
+	char text[50];
+	// Get the text from the type column (4).
+	ListView_GetItemText(hwnd, focused, 4, text, sizeof(text));
+	if (strcmp(text, "Note") != 0) {
+		return;
+	}
+	fill(begin(text), end(text), 0);
+	MidiNote note;
+	// Get the text from the position column (1).
+	ListView_GetItemText(hwnd, focused, 1, text, sizeof(text));
+	// Convert this to project time. text is always in measures.beats.
+	note.start = parse_timestr_pos(text, 2);
+	fill(begin(text), end(text), 0);
+	// Get the text from the length column (2).
+	ListView_GetItemText(hwnd, focused, 2, text, sizeof(text));
+	// Convert this to project time. text is always in measures.beats.
+	auto length = parse_timestr_len(text, 0, 2);
+	note.end = note.start + length;
+	fill(begin(text), end(text), 0);
+	// Get the text from the channel column (3).
+	ListView_GetItemText(hwnd, focused, 3, text, sizeof(text));
+	note.channel = atoi(text);
+	fill(begin(text), end(text), 0);
+	// Get the text from the parameter (note name) column (5).
+	ListView_GetItemText(hwnd, focused, 5, text, sizeof(text));
+	string noteNameWithOctave { text };
+	static const string noteNames[] = {"C", "C#", "D", "D#", "E", "F",
+		"F#", "G", "G#", "A", "A#", "B"};
```

Comment:
Note names can be customised, so this is going to fail in that case. See OSARA's getMidiNoteName function and its use of GetTrackMIDINoteName. I don't really know if there's a way to deal with that here?

Reply:
Ugh, problem here is that we're transforming from note name to pitch.
I've also considered finding a way to get the right note event and just use that to play the note, but I"m afraid we can only iter through the list and count all the notes, which is probably pretty slow.

Reply:
Hmm. I wonder if Cockos would be willing to give us an API function to convert event list view indexes to note/CC indexes?

In the meantime, I think we can still go ahead with this, since it's still useful in cases (probably the majority) where there aren't custom note names.

Reply:
> Hmm. I wonder if Cockos would be willing to give us an API function to convert event list view indexes to note/CC indexes?

Either that, or a function to convert custom note names back to pitch.